ui/DesignSystem/Component/Sidebar.svelte

```diff
@@ -178,5 +172,11 @@
         <Title>Profile</Title>
       </div>
     </li>
+
+    <li class="item">
```

Comment:
Pretty sure this is actually where the 'new project' button will go in the sidebar; just using it as a convenience for now